src/js/tabs/configuration.js

```diff
@@ -529,7 +529,8 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         // generate GPS
         var gpsProtocols = [
             'NMEA',
-            'UBLOX'
+            'UBLOX',
+            'MSP'
```

Comment:
This will need to be version gated so that it is only available for API versions 1.41 and up.

Reply:
How about that:
```
        var gps_protocol_e = $('select.gps_protocol');
        var gpsProtocolsLength = 2;
        if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
            gpsProtocolsLength = 3;
        }
        for (var i = 0; i < gpsProtocolsLength; i++) {
            gps_protocol_e.append('<option value="' + i + '">' + gpsProtocols[i] + '</option>');
        }
```

Reply:
Usually, if version bigger than X, we append the new element to the combo. But your solution is not bad neither.